messaging/rabbit.go

```diff
@@ -204,11 +206,20 @@ func ValidateRabbitConfigStruct(servers []string, exchange ExchangeDefinition, q
 
 // ConnectToRabbit will open a TLS connection to rabbit mq
 func ConnectToRabbit(config *RabbitConfig, log *logrus.Entry) (*Consumer, error) {
-	if err := ValidateRabbitConfig(config); err != nil {
+	var err error
```

Comment:
declare this in the `if` block you're using it in

Reply:
The problem I found is, when I assign the result of discoverRabbitServers to servers and err, if I use :=, it fails because of:
```
servers declared and not used (build)
```
I guess I could have used a tmp var and then assign it to servers?